modules/tinymce/src/themes/mobile/main/ts/ui/SerialisedDialog.ts

```diff
@@ -120,14 +120,14 @@ const sketch = function (rawSpec) {
           }),
           Keying.config({
             mode: 'special',
-            focusIn (dialog/*, specialInfo */) {
+            focusIn(dialog/* , specialInfo */) {
```

Comment:
This comment spacing is a bit weird - I could change this to `focusIn(dialog, _specialInfo) {` to get it to ignore the unused variable.

Reply:
I converted all commented out parameters into either unused parameters (using leading underscore) or I removed them.